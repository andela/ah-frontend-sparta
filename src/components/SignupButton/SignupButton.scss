@import '../../assets/scss/shared.scss';

button {
    padding: rem(10) rem(30);
    border-radius: rem(25);
    font-size: rem(14);
    font-weight: bold;
    outline: transparent;

    &:active {
        border-color: $primary-color;
    }
}
