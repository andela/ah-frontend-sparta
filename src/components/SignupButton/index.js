import React from 'react';

import './SignupButton.scss';

const SignButton = ({ value }) => {
  return <button>{value}</button>;
};

export default SignButton;
