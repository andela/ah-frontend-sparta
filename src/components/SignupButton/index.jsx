import React from 'react';
import PropTypes from 'prop-types';

import './SignupButton.scss';

const SignButton = ({ value, handleSubmit }) => <button id="loginButton" type="submit" onClick={handleSubmit}>{value}</button>;

SignButton.propTypes = {
  value: PropTypes.string,
  handleSubmit: PropTypes.func,
};

SignButton.defaultProps = {
  value: '',
  handleSubmit: () => {},
};

export default SignButton;
