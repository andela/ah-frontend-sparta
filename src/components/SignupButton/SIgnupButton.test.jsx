import React from 'react';
import renderer from 'react-test-renderer';
import SignupButton from './index';

test('snapshot test', () => {
  const wrapper = renderer.create(<SignupButton />);
  expect(wrapper).toMatchSnapshot();
});
