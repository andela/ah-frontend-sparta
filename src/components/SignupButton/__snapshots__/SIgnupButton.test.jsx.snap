// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<button
  type="submit"
>
  
</button>
`;
